pkgs/build-support/fetchmavenartifact/default.nix

```diff
@@ -3,9 +3,8 @@
 { fetchurl, stdenv }:
 let
   defaultRepos = [
-    "http://repo1.maven.org/maven2"
+    "https://repo1.maven.org/maven2"
     "http://oss.sonatype.org/content/repositories/releases"
-    "http://oss.sonatype.org/content/repositories/public"
```

Comment:
it's not accessible for listing on that level, but works with explicit paths, e.g.:
https://oss.sonatype.org/content/repositories/public/io/github/opencubicchunks/cubicchunks/1.12.2-0.0-SNAPSHOT/ or https://oss.sonatype.org/content/repositories/public/org/codehaus/gant/gant_groovy2.3/1.9.12-SNAPSHOT/
so please revert it back, unless it causes issues (if so, please elaborate more)